app/views/katello/api/v2/activation_keys/base.json.rabl

```diff
@@ -26,8 +26,8 @@ child :content_view_environments => :content_view_environments do
       lifecycle_environment_library: cve.lifecycle_environment&.library?
     }
   end
-  node :candlepin_name do |cve|
```

Comment:
I 'd be nervous to do this for an api response. Can we may be have both ?

Reply:
The code that introduced this API response node was just merged today IIRC, so I don't think anyone relies on it yet. Thoughts?